client/src/pages/private/ParticipantDetailsView.js

```diff
@@ -304,7 +304,17 @@ export default () => {
               initialValues={actualParticipant}
               validationSchema={EditParticipantFormSchema}
               onSubmit={onUpdateInfo}
-              onClose={() => setShowEditModal(false)}
+              onClose={() => {
```

Comment:
Looks like I missed an instance of `EditParticipantForm` 🤔 heads up that `validationSchema` and `onSubmit` aren't used by the component anymore, if this page uses the same component but has a different edit function `EditParticipantForm` might need to be looked at again.

Do you know if it's intentional that these two pages have different onSubmits?